bin/rename_project

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Replaces project name throughout the mix project
+#
+# $ bin/rename_project NewProjectName new_project_name
+
+CURRENT_MODULE_NAME="Nimble"
+CURRENT_OTP_NAME="nimble"
+
+echo "Renaming from $CURRENT_MODULE_NAME, $CURRENT_OTP_NAME to $1, $2\n"
+read -p "ARe you sure? (Press y to confirm)" -n 1 -r
+if [[ ! $REPLY =~ ^[Yy]$ ]]
+then
+    exit 1
+fi
+
+find . ! -path "./deps/*" -name "*.ex*" -print0 | xargs -0 sed -i '' -e "s/$CURRENT_MODULE_NAME/$1/g"
+find . ! -path "./deps/*" -name "*.ex*" -print0 | xargs -0 sed -i '' -e "s/$CURRENT_OTP_NAME/$2/g"
+
+mv lib/$CURRENT_OTP_NAME.ex lib/$2.ex
+mv lib/${CURRENT_OTP_NAME}_web.ex lib/$2_web.ex
+mv lib/$CURRENT_OTP_NAME lib/$2
+mv lib/${CURRENT_OTP_NAME}_web lib/$2_web
+[ -d test/${CURRENT_OTP_NAME} ] && mv test/${CURRENT_OTP_NAME} test/$2
+[ -d test/${CURRENT_OTP_NAME}_web ] && mv test/${CURRENT_OTP_NAME}_web test/$2_web
+
+echo "\n\nIf you cloned this template from Github, you may want to reinitialize git:"
+echo "    rm -rf .git/"
+echo ""
```

Comment:
@Tolsee Please rename the docker container which is merged in https://github.com/nimblehq/elixir-templates/pull/9  too 👍